### PR TITLE
add example usage of social profiles

### LINKED ISSUE #138: Specifying a list of social profiles — usage explanation
Can a usage example please be provided for specifying a list of social profiles.

I've tried adding both `social: links:` and just `links:` but neither output anything.

### PR DESCRIPTION
**Pull request by DirtyF**
fix #138

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,18 @@ The SEO tag will respect any of the following if included in your site's `_confi
 * `social` - For [specifying social profiles](https://developers.google.com/structured-data/customize/social-profiles). The following properties are available:
   * `name` - If the user or organization name differs from the site's name
   * `links` - An array of links to social media profiles.
+  
+  ```yml
+  social:
+  name: Ben Balter
+  links:
```
Comment (pathawks) [inline README.md]: Shouldn't `name` and `links` be indented?

Comment (DirtyF) [inline README.md]: ➡️   done. Thanks @pathawks

**Comment by benbalter** [comment]
@jekyllbot: merge +docs

**Comment by benbalter** [comment]
Thanks, as always, @DirtyF!

**Pull request merged**